syntax = "proto3";

package executor;

service ExecutorService {
  
}
